tests/components/stream/test_hls.py

```diff
@@ -185,7 +186,7 @@ async def test_hls_stream(
     assert stream.get_diagnostics() == {
         "container_format": "mov,mp4,m4a,3gp,3g2,mj2",
         "keepalive": False,
-        "orientation": 1,
+        "orientation": Orientation.NO_TRANSFORM,
```

Comment:
There are a few instances like this which are really cleanup from #81835. They can be moved out into a separate PR if necessary.